bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/DescriptorRpc.scala

```diff
@@ -0,0 +1,32 @@
+package org.bitcoins.rpc.client.common
+
+import org.bitcoins.rpc.jsonmodels.{
+  DeriveAddressesResult,
+  GetDescriptorInfoResult
+}
+import play.api.libs.json.{JsString, Json}
+import org.bitcoins.rpc.serializers.JsonSerializers._
+
+import scala.concurrent.Future
+
+/**
+  * RPC calls in V18 that use descriptor to give us output information
+  * @see [[https://bitcoincore.org/en/doc/0.18.0/rpc/util/deriveaddresses/]]
+  * @see [[https://bitcoincore.org/en/doc/0.18.0/rpc/util/getdescriptorinfo/]]
+  */
+trait DescriptorRpc {
+  self: Client =>
+
+  def deriveAddresses(
+      descriptor: String,
+      range: Option[Vector[Double]]): Future[DeriveAddressesResult] = {
+    bitcoindCall[DeriveAddressesResult](
+      "deriveaddresses",
+      List(JsString(descriptor), Json.toJson(range)))
+  }
+
+  def getDescriptorInfo(descriptor: String): Future[GetDescriptorInfoResult] = {
```

Comment:
tests for these two

Reply:
They already exist in `BitcoindV18ClientTest`